site/css/_colors.css

```diff
@@ -9,7 +9,8 @@
 */
 
 /* Main Colour Palette */
-@value badgerRed: #FC1D42;
+@value badgerRed: #FC3A54;
+@value badgerRedOnBlack: #E5051A;
```

Comment:
badgerRed and badgerRedOnBlack values need to be switched throughout the site - my mess up in the comment on the original bug mixing these up